apps/condo/domains/billing/schema/BillingAccount.test.js

```diff
@@ -2,404 +2,467 @@
             })
-
-            const { context: context2 } = await makeContextWithOrganizationAndIntegrationAsAdmin()
-            const [property2] = await createTestBillingProperty(admin, context2)
-            const [billingAccount2] = await createTestBillingAccount(admin, context2, property2, {
-                globalId: 'cat',
+            describe('Multiple objects', () => {
+                test('Admin can for any context', async () => {
+                    const [accounts] = await createTestBillingAccounts(admin,
+                        [context, anotherContext],
+                        [property, anotherProperty])
+                    expect(accounts).toBeDefined()
+                    expect(accounts).toHaveLength(2)
+                    expect(accounts).toEqual(expect.arrayContaining([
+                        expect.objectContaining({
+                            context: expect.objectContaining({ id: context.id }),
+                            property: expect.objectContaining({ id: property.id }),
+                        }),
+                        expect.objectContaining({
+                            context: expect.objectContaining({ id: anotherContext.id }),
+                            property: expect.objectContaining({ id: anotherProperty.id }),
+                        }),
+                    ]))
+                })
+                test('Support cannot', async () => {
+                    await expectToThrowAccessDeniedErrorToObjects(async () => {
+                        await createTestBillingAccounts(support, [context], [property])
+                    })
+                })
+                describe('User', () => {
+                    describe('Integration account can if linked to permitted integration and property via context', () => {
+                        test('All permitted contexts should pass', async () => {
+                            const [accounts] = await createTestBillingAccounts(integrationUser,
+                                [context, context],
+                                [property, property])
+                            expect(accounts).toBeDefined()
+                            expect(accounts).toHaveLength(2)
+                            expect(accounts[0]).toEqual(expect.objectContaining({
+                                context: expect.objectContaining({ id: context.id }),
+                                property: expect.objectContaining({ id: property.id }),
+                            }))
+                            expect(accounts[1]).toEqual(expect.objectContaining({
+                                context: expect.objectContaining({ id: context.id }),
+                                property: expect.objectContaining({ id: property.id }),
+                            }))
+                        })
+                        test('Partly permitted must fail', async () => {
+                            await expectToThrowAccessDeniedErrorToObjects(async () => {
```

Comment:
You also need to check the database state if that case. Is it create some object or not?

Reply:
You're right. Will fix this 👍

Reply:
not resolved yet 😄

Reply:
Added case in `BillingProperty` tests. 

Not added to another models since it's same behaviour across all models and only depends on Keystone